src/main/java/br/unb/cic/js/walker/RepositoryWalkerTask.java

```diff
@@ -69,19 +65,14 @@ public void run() {
                 errorsWriter.flush();
             } catch (IOException ex) {
                 logger.error("failed to write on report/errors file for project {}", walker.project);
-                throw new RuntimeException(ex);
             }
 
-            synchronized (results) {
-                results.write(content.toString());
-                results.flush();
-            }
         } catch (IOException ex) {
             logger.error("failed to create a report/errors file for project {}", walker.project);
-            throw new RuntimeException(ex);
         } catch (Exception ex) {
+            ex.printStackTrace();
+            System.out.println(ex.getMessage());
```

Comment:
don't use `System.out.println`, use the logger with a replacement variable such as the one in line 83.

Example: `logger.error("failed to traverse project {}, reason {}", walker.project, ex.getMessage());`